.ci/ibdev2netdev

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Use `/bin/sh -eE` where it's possible or `/bin/bash -eE` if you use bash specific constructions

Reply:
Done